cmake/scripts/common/AddonHelpers.cmake

```diff
@@ -255,8 +257,7 @@ macro (build_addon target prefix libs)
 
         if(CMAKE_BUILD_TYPE MATCHES Debug)
           # for debug builds also install the PDB file
-          get_filename_component(LIBRARY_DIR ${LIBRARY_LOCATION} DIRECTORY)
-          install(FILES ${LIBRARY_DIR}/${target}.pdb DESTINATION ${target}
+          install(FILES ${PROJECT_BINARY_DIR}/${CMAKE_BUILD_TYPE}/${target}.pdb DESTINATION ${target}
                   COMPONENT ${target}-${${prefix}_VERSION})
         endif()
```

Comment:
Sorry if I wasn't clear enough. When I tried to package a binary addon with the CMake generator -G"Visual Studio 14" as debug or release config the zip file generation fails. With my patch all is fine. It needs a further NMake test to see if this also works. Will try that at the weekend.